vcluster_versioned_docs/version-0.20.0/configure/vcluster-yaml/policies/resource-quota.mdx

```diff
@@ -37,10 +37,9 @@ policies:
   resourceQuota:
     enabled: true
     quota:
-      hard:
-        cpu: "10"
-        memory: 20Gi
-        pods: "10"
+      limits.cpu: "10"
```

Comment:
With this I get error - `	* ResourceQuota "vc-resourcequota2" is invalid: [spec.hard[limits.pods]: Invalid value: limits.pods: must be a standard resource type or fully qualified, spec.hard[limits.pods]: Invalid value: limits.pods: must be a standard resource for quota]`

I think it should be 
```
policies:
  resourceQuota:
    enabled: true
    quota:
       cpu: "10"
       memory: 20Gi
       pods: "10"
    scopeSelector:
      matchExpressions:
        - operator : In
          scopeName: PriorityClass
          values: ["medium"]
```

Reply:
It needs to be:
```
policies:
  resourceQuota:
    enabled: true
    quota:
      limits.cpu: "10"
      limits.memory: 20Gi
      count/pods: "10"
    scopeSelector:
      matchExpressions:
        - operator : In
          scopeName: PriorityClass
          values: ["medium"]
```

Since pods is a quantified resource, it needs `count/` in front of it.